### ISSUE TITLE
Typescript - EventEmitter class extension missing

### ISSUE THREAD

**Issue by joshmeads** [darkroomengineering/lenis#53]
**Describe the bug**
Currently the types are getting generated without the class extension from `tiny-emitter` causing typescript to error.

```ts
lenis.on('scroll', handleScroll); // Property 'on' does not exist on type 'Lenis'.
```

Current type output:
```ts
export default class Lenis {}
```

Expected type output (matches implementation):
```ts
export default class Lenis extends EventEmitter {}
```

I've tried solving this myself however I'm not very familiar with JSDoc syntax and couldn't get `@typedef {import('tiny-emitter')} EventEmitter` to play nicely with `@extends EventEmitter`. 

If it would help I'd be happy to spend the time to migrate it directly to typescript, I didn't want to spend the time without any input from yourselves first.

**Comment by clementroche** [comment]
Hi, we don't want to migrate to Typescript because we don't have any experience with it and we don't see the benefits. So you can PR if you find the fix.

**Comment by joshmeads** [comment]
Understood, thanks for the swift reply! As far as I've read there isn't a way to do this. I'll likely fork for now and if I learn of a solution will open a PR for sure.

**Comment by DaniAcu** [comment]
Hi! I created a PR to fix that. You just need to import with named import to get also the types, because tiny import just export the type in that way.

```ts
import { TinyEmitter } from "tiny-emitter";
```

**Comment by joshmeads** [comment]
> fix: types for event emitter #57

Seriously it was that simple? Here i was pulling my hair out trying JSDoc import syntax for hours 😆 

@DaniAcu thank you! Always great when you can learn something new
